system_intelligence/ram_info.py

```diff
@@ -1,190 +1,229 @@
-"""Functions to query system's RAM."""
-
 import logging
 import os
 import subprocess
 import typing as t
 from xml.etree import ElementTree as ET
-
 import psutil
-from rich.console import Console
 from rich import print
+from sys import platform
 
+from .base_info import BaseInfo
 from .util.process_util import is_process_accessible
-from .util.rich_util import create_styled_table
 from .util.unit_conversion_util import bytes_to_hreadable_string, hz_to_hreadable_string
 
 _LOG = logging.getLogger(__name__)
 
 try:
     import pyudev
-
     pyudev.Context()
```

Comment:
On MacOS actually, `pyudev` seems not available. It catches the `ImportError` but I get an Info along the lined `Exception ignore in some pyudev file AttributeError`. It does not cause the program to fail but its rather annoying. I tried to catch it as well but this did not work. I also tried a nested `try except` in the first `except` statement but this completly prohibits any printing ;(.

Any ideas how to fix this @Zethson ?